pontoon/teams/views.py

```diff
@@ -122,13 +122,11 @@ def ajax_permissions(request, locale):
     translators = l.translators_group.user_set.exclude(pk__in=managers).all()
     all_users = User.objects.exclude(pk__in=managers).exclude(pk__in=translators).exclude(email='')
 
-    contributors = (
-        User.translators
-        .filter(translation__locale=l)
-        .values_list('email', flat=True)
-        .distinct()
-    )
+    contributors_qs = User.translators.with_translation_counts(None, Q(locale=l))
+    contributors = set([contributor.email for contributor in contributors_qs])
```

Comment:
no need for brackets here you can you a generator expression rather than list comprehension

Reply:
also might be more efficient to add `.iterator()` on the qs

Reply:
Updated. `.iterator()` I cannot use because `with_translation_counts` returns a `list`.